src/components/VisualizationPlugin/VisualizationPlugin.js

```diff
@@ -41,6 +41,26 @@ export const VisualizationPlugin = ({
     const [contextualMenuConfig, setContextualMenuConfig] = useState({})
     const [showLegendKey, setShowLegendKey] = useState(false)
     const [renderId, setRenderId] = useState(id)
+    const [size, setSize] = useState({ width: 0, height: 0 })
+
+    const containerRef = useCallback((node) => {
```

Comment:
Perhaps since this is a [callback ref](https://legacy.reactjs.org/docs/refs-and-the-dom.html#callback-refs), and not just a regular reference to a DOM element, it could be better to reflect this in the variable name, i.e. `containerCallbackRef` of `onContainerMounted` (not so sure about that one)